### DISCUSSION TITLE
Mozilla CA Certificates are outdated

### DISCUSSION THREAD

**Discussion by g-andrade** [curl/curl#15636] [General]
Hi,

Thanks for providing the [CA extract](https://curl.se/docs/caextract.html)! I poll it once a day.

As mentioned in an [earlier discussion](https://github.com/curl/curl/discussions/13100), updates to it are automated by a cronjob. This cronjob appears to not have been working lately, since there are updates upstream that haven't showed up on the CA extract webpage.

In any case, I should switch to using the mk-ca-bundle script (so simple to use!), so this post is more of a heads up than anything else.

**Comment by jay** [comment]
Please take further discussion to curl/curl-www#409
